sqlx-macros/src/derives/mod.rs

```diff
@@ -33,5 +33,6 @@ pub(crate) fn rename_all(s: &str, pattern: RenameAll) -> String {
         RenameAll::LowerCase => s.to_lowercase(),
         RenameAll::SnakeCase => s.to_snake_case(),
         RenameAll::UpperCase => s.to_uppercase(),
+        RenameAll::ScreamingSnakeCase => s.to_shouty_snake_case(),
```

Comment:
We could also name this enum (and the text option for it) `ShoutySnakeCase` to more closely match this library. Happy to change this, either way!